backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/button/BpkButtonIconPosition.kt

```diff
@@ -0,0 +1,8 @@
+package net.skyscanner.backpack.compose.button
+
+enum class BpkButtonIconPosition {
```

Comment:
i'm a big fan of keeping enum definitions in the same file as where they are used to make it easier to see what the different properties are, rather than having to open lots of little files